module/x/peggy/spec/04_messages.md

```diff
@@ -82,83 +65,59 @@ This message will fail if:
 - Failure to build a batch of transactions.
 - If the orchestrator address is not present in the validator set
 
-### MsgConfirmBatch
-
-When a `MsgRequestBatch` is observed, validators need to sign batch request to signify this is not a maliciously created batch and to avoid getting slashed. 
+### MsgSubmitConfirm
```

Comment:
This should be `MsgConfirmEthereumEvent`

Reply:
was this changed in your work?